src/routes/login/+page.server.ts

```diff
@@ -0,0 +1,14 @@
+import { redirect } from "@sveltejs/kit";
+import { getOIDCAuthorizationUrl, getRedirectURI } from "$lib/server/auth";
+
+export const actions = {
+	default: async function ({ url, locals }) {
+		// TODO: Handle errors if provider is not responding
+		const ssoAuthorizationUrl = await getOIDCAuthorizationUrl(
+			{ redirectURI: getRedirectURI(url) },
+			{ sessionId: locals.sessionId }
+		);
+
+		throw redirect(303, ssoAuthorizationUrl);
```

Comment:
```suggestion
		const authorizationUrl = await getOIDCAuthorizationUrl(
			{ redirectURI: getRedirectURI(url) },
			{ sessionId: locals.sessionId }
		);

		throw redirect(303, authorizationUrl);
```

nit: this isn't really sso (there's also  a `SSOUserInformation` type)